### PR TITLE
parse: normalise crlf for windows

### PR DESCRIPTION
**Pull request by boranx**
Fixes: #142

### CODE REVIEW

**Comment by jpreese** [comment]
If we're having to parse the input in the test, outside of the core logic of the parser, to get the test to pass, it seems like the normalization you added should just be added to the method itself.

File: commands/parse/parse_test.go [hunk 1]
```diff
@@ -81,9 +82,11 @@ func TestInputFlagForparseInput(t *testing.T) {
 		viper.Reset()
 		viper.Set("input", testunit.input)
 		parsed, _ := parseInput(ctx, testunit.fileList)
+		tmp := string(parsed)
+		tmp = strings.Replace(tmp, "\\r", "", -1)
```
Comment (jpreese) [inline commands/parse/parse_test.go]: Maybe we should just add this to `parseInput` ?

Comment (boranx) [reply commands/parse/parse_test.go]: I got the perspective you mentioned. I'm going to add this to the method itself.

File: commands/parse/parse.go [hunk 2]
```diff
@@ -68,7 +77,13 @@ func parseInput(ctx context.Context, fileList []string) ([]byte, error) {
 	return bundle, nil
 }
 
+func finalize(input []byte) string {
```
Comment (jpreese) [inline commands/parse/parse.go]: I dont really think we need this method. In #152 I just make `parseInput` return a string so we can avoid the `string()` casting after getting the value. So in this case, we could just have `parseInput` itself do the endline normalization and return a string.

Comment (boranx) [reply commands/parse/parse.go]: umm, then you can add:
- ```output = strings.Replace(output, "\\r", "", -1)```
to pass the tests in #152, and I can close this one.

Comment (jpreese) [reply commands/parse/parse.go]: Works for me! This line is added in https://github.com/instrumenta/conftest/pull/152/commits/be382cf90abaa9b2985cabb077184919860ccf99

Comment (boranx) [reply commands/parse/parse.go]: cool! thanks

**Comment by boranx** [comment]
Closing this because #152 contains this.

**Pull request closed without merging**